source/code/plugins/out_oms_blob.rb

```diff
@@ -29,7 +29,7 @@ def initialize
     config_param :key_path, :string, :default => '/etc/opt/microsoft/omsagent/certs/oms.key'
     config_param :blob_uri_expiry, :string, :default => '00:10:00'
     config_param :url_suffix_template, :string, :default => "custom_data_type + '/00000000-0000-0000-0000-000000000002/' + OMS::Common.get_hostname + '/' + OMS::Configuration.agent_id + '/' + suffix + '.log'"
-    config_param :proxy_conf_path, :string, :default => '/etc/opt/microsoft/omsagent/conf/proxy.conf'
+    config_param :proxy_conf_path, :string, :default => '/etc/opt/microsoft/omsagent/proxy.conf'
```

Comment:
This change also needs to be added to source/code/plugins/in_heartbeat_request.rb and installer/conf/omsagent.d/heartbeat.conf.
Currently, heartbeat.conf has proxy_path listed as %CONF_DIR_WS%/proxy.conf ; for minimal changes, this should simply be removed and change in_heartbeat_request.rb to have the same config param as the out_oms_* files now do.